azure-pipelines.yml

```diff
@@ -22,16 +22,16 @@ jobs:
     displayName: 'Use Python $(python.version)'
 
   - script: |
-      python -m pip install --upgrade pip
+      python -m pip install --upgrade pip wheel
     displayName: 'Install dependencies'
 
   - script: |
       python -m pip install .
     displayName: 'Build project'
 
   - script: |
-      pip install pytest pytest-azurepipelines
-      pytest test/
+      pip install pytest pytest-azurepipelines pytest-xdist
```

Comment:
shouln't it be `pytest-xdist[psutil]` to enable the detection of the number of CPU?

Reply:
Oh, I guess we can do that although it detects the number of CPUs anyway.

Reply:
The answer seems to be, no we don't need it since we've got `os.sched_getaffinity()` in all supported Python versions we run pytest with.

psutil is only required if we run pytest on the codebase using Python 2.7.